src/lpc.jl

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Yes. The two algorithms can live in a single LPC module, but since the functionality is independent of the rest of DSP.jl, it should be a separate module.